modules/administration/pages/virtual-hosts.adoc

```diff
@@ -111,4 +111,10 @@ If you do not refresh the data from a new Virtual Host Manager, host data will n
 ====
 +
 
-. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems] .
+. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems].
+
+[NOTE]
+====
+Depending on your environment and settings, you might receive an ``invalid certificate`` error when you connect to the server.
+If this occurs, check that your SSL certificate is current, run the [command]``update-ca-certificates`` command on the command line to re-trust your certificate, and restart the spacewalk services.
```

Comment:
I know that @mcalmer suggested to avoid precise instructions here. Still, I would prefer something more explicit:
```
check that your SSL certificate is current
```
=>
```
extract the certificate from the ESXi server and copy it into `/etc/pki/trust/anchors`
```